README.md

```diff
@@ -63,7 +112,178 @@ And finally you can start the VM:
+ App name            | scribe
+ Code                | [charges-scribe](https://github.com/LandRegistry/charges-scribe)
+ Infrastructure code | [charges-scribe/puppet/scribe](https://github.com/LandRegistry/charges-scribe/tree/master/puppet/scribe)
+ Development Host    | http://scribe.dev.service.gov.uk
+ Code on VM          | /opt/scribe
+ Default port        | 0.0.0.0:9010
+
+### Verifudge
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | verifudge
+ Code                | [charges-verifudge](https://github.com/LandRegistry/charges-verifudge)
+ Infrastructure code | [charges-verifudge/puppet/verifudge](https://github.com/LandRegistry/charges-verifudge/tree/master/puppet/verifudge)
+ Development Host    | http://verifudge.dev.service.gov.uk
+ Code on VM          | /opt/verifudge
+ Default port        | 0.0.0.0:9080
+
+##Development usage
+
+### Development path
+
+The VM uses rsync to mirror the path you set when installing:
+```bash
+export development=/path/to/your/code
+```
+
+This folder will exist in `~/development`. In this way you can keep using your
+IDE or editor of choice while benefitting from the Dev VM.
+
+### Controlling apps
+
+Every app is controlled through systemd. They each expose a service that can be
+used to do a variety of tasks. They also expose an nginx config that can be used
+in development to speed up the connecting of services together.
+
+The following instructions assume you have sshed in to a vm built using a clean
+copy of this repo.
+
+#### Starting / Stopping / Restarting an app
+
+To start / stop / restart any app use:
+
+```
+> sudo systemctl <action> -u <app_name>
+```
+
+Where <app_name> is the name listed in the [Apps installed](#apps-installed)
+tables above and action is any of `start`, `stop`, `restart`.
+
+**Examples**
+
+To start the Borrower Frontend use: `sudo systemctl start -u borrower_frontend`
+
+To stop the Verifudge use: `sudo systemctl stop -u verifudge`
+
+To restart the Case API use: `sudo systemctl restart -u case_api`
+
+*note:* you can also use the old initd syntax since initd passes through to systemd
+on centos, e.g. `sudo service borrower_frontend start`
+
+#### Replacing an app with your development version
+
+A useful part of having all the apps installed and configured through nginx
+before developing is that you can easily make your changes to any app and then
+piggy back on the nginx config to quickly integrate your changes into the full
+system.
+
+**Example**
+
+Lets say I'm working on a change for the Borrower Frontend. First I'll provision
+the Dev VM. The Borrower Frontend that is on master will be installed and exposed
+on http://borrower_frontend.dev.service.gov.uk and all other apps will be
+configured to connect to it, including the acceptance tests.
+
+Next I cd into `~/developement/borrower_frontend` where I have the Borrower
```

Comment:
Should be ~development/borrower_frontend? i.e. only three e's in development? Sorry for being picky but as it's something people might likely copy and paste it might get annoying =)